Source/Csla.AspNetCore/Blazor/ActiveCircuitState.cs

```diff
@@ -0,0 +1,25 @@
+﻿//-----------------------------------------------------------------------
```

Comment:
Thank you for all the testing @swegele - hopefully this latest commit fits into what you discovered?

Reply:
Wunder Bar!  This little trick seems like a nice pattern to have in the back pocket...thanks to you both.

Reply:
I was thinking the same - a nice pattern to keep in mind for the future.